demo/mmediting_inference_tutorial.ipynb

```diff
@@ -0,0 +1,346 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# MMEditing Inference Tutorial\n",
+    "\n",
+    "Welcome to MMEditing! In this tutorial you will learn how to use MMEditing inference api to predict your own image or video. \n",
+    "\n",
+    "This is a quick guide for you to infer with existing models. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Install MMEditing\n",
+    "Please refer to [README.md](README.md) for installation instruction."
+   ]
+  },
+  {
+   "cell_type": "markdown",
```

Comment:
You can also provide the script to download resources in `demo/readme.md`

Reply:
resources need a url :crystal_ball: